www/src/pages/en/usage/first-steps.md

```diff
@@ -0,0 +1,31 @@
+---
+title: First Steps
+description: Getting started with your new T3 App
+layout: ../../../layouts/docs.astro
+---
+
+You just scaffolded a new T3 App and are ready to go. Here is the bare minimum to get your app working.
+
+## Database
+
+If your app includes Prisma, make sure to run `npx prisma db push` from the root directory of your app. Note that you need to restart the dev server after doing this.
```

Comment:
```suggestion
If your app includes Prisma, make sure to run `npx prisma db push` from the root directory of your app. This command will sync your Prisma schema with your database and will generate the TypeScript types for the Prisma client based on your schema. Note that you need to restart the TypeScript server after doing this so that it can detect the generated types.
```